zipcodes/__init__.py

```diff
@@ -24,11 +24,21 @@
 __email__ = "pianka@eml.cc"
 __license__ = "MIT"
 __package__ = "zipcodes"
-__version__ = "1.1.2"
+__version__ = "1.1.2a"
```

Comment:
Can you change this to 1.1.3? I think this warrants a patch version bump.

Reply:
This requested change has been made

Reply:
Sean - just want to validate that you can see the version change - I committed the change - but not sure if I did it correctly - so just anted to validate you can see the 2nd change I made.

Reply:
Yes, I see it in the Python file. There is also a file which contains the version separately, could you update this to 1.1.3 as well? That's all left before I can publish a new version.

Thanks for your PR.